src/Comments.jsx

```diff
@@ -47,20 +47,20 @@ class _Comments extends React.Component {
 
     componentDidMount() {
         this._ismounted = true;
-        this.customizrClient.fetchSettings().then(json => {
-            this.setState({
-                alertDismissed: json.mentionsUsageNotification &&
-                    json.mentionsUsageNotification.alertDismissed === true,
-                selectedAccessibilityOption: this.state.commentAccessibilityLevels.find(l => l.value === json.selectedAccessibility)
-            }, () => { this.resetSelectedAccessibilityOption(); });
-        })
     }
 
     componentWillUnmount() {
         this._ismounted = false;
     }
 
     componentWillReceiveProps(newProps) {
```

Comment:
The first time that the component mounts componentWillReceiveProps is not called.

```
    componentDidMount() {
this.componentWillReceiveProps(this.props)
     }
```

Reply:
I tested and it did get called. Can you explain it?

Reply:
I think there seems to be immediately a second render or something ...